pkg/cli-runtime/config_test.go

```diff
@@ -153,3 +153,51 @@ func TestConfig_Print(t *testing.T) {
 		})
 	}
 }
+func TestConfig_EmojiPrint(t *testing.T) {
+	scheme := runtime.NewScheme()
+	config := cli.NewDefaultConfig("test", scheme)
+
+	tests := []struct {
+		name    string
+		format  string
+		icon    cli.Icon
+		args    []interface{}
+		printer func(icon cli.Icon, format string, a ...interface{}) (n int, err error)
+		stdout  string
+		stderr  string
+	}{{
+		name:    "EmojiSuccessf",
+		format:  "%s",
+		icon:    cli.FloppyDisk,
+		args:    []interface{}{"Source"},
```

Comment:
update args to some other variable in one of the tests

Reply:
Done